src/Orchard.Cms.Web/Modules/Orchard.OpenId/JwtBearerConfigureOptions.cs

```diff
@@ -33,7 +33,7 @@ public void Configure(JwtBearerOptions jwtBearerOptions)
                 jwtBearerOptions.AutomaticAuthenticate = true;
                 jwtBearerOptions.AutomaticChallenge = true;
                 jwtBearerOptions.RequireHttpsMetadata = !openIdSettings.TestingModeEnabled;
-                jwtBearerOptions.Audience = openIdSettings.Audience;
+                jwtBearerOptions.Audience = openIdSettings.Audiences!=null?string.Join(",",openIdSettings.Audiences):null;
```

Comment:
Not sure if I tested it works. The point is I didn't know that way of assigning ValidAudiences exist and it looks the right one. I'm going to change it.